src/locales/de-DE.json

```diff
@@ -8,6 +8,19 @@
         "required": "Die Administrationsebene muss ausgewählt werden."
       }
     },
+    "befristung": {
+      "assignBefristung": "Befristung zuordnen",
+      "befristung": "Befristung",
+      "rules": {
+        "required": "Die Befristung muss eingegeben werden.",
+        "format": "Die eingegebene Befristung hat nicht das richtige Format DD.MM.JJJJ.",
+        "pastDateNotAllowed": "Datum in der Vergangenheit ist nicht erlaubt."
+      },
+      "selectBefristung": "Befristung auswählen",
+      "unlimited": "Unbefristet",
```

Comment:
Können wir das in den Radio Buttons groß geschrieben lassen und im success template klein schreiben? Florentina ist auch dafür.

Reply:
Passt